lms/djangoapps/ccx/migrations/0004_seed_forum_roles_in_ccx_courses.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from ccx_keys.locator import CCXLocator
+from courseware.courses import get_course_by_id
+from django.db import migrations
+from django.http import Http404
+
+from django_comment_common.models import FORUM_ROLE_ADMINISTRATOR, FORUM_ROLE_MODERATOR, \
+    FORUM_ROLE_COMMUNITY_TA, FORUM_ROLE_STUDENT
+from django_comment_common.utils import STUDENT_ROLE_PERMISSIONS,  MODERATOR_ROLE_PERMISSIONS, \
+    ADMINISTRATOR_ROLE_PERMISSIONS
+
+
+def seed_forum_roles_for_existing_ccx(apps, schema_editor):
+    """
+    Seed forum roles and make CCX coach forum admin for respective CCX(s).
+
+    Arguments:
+        apps (Applications): Apps in edX platform.
+        schema_editor (SchemaEditor): For editing database schema i.e create, delete field (column)
+
+    """
+    CustomCourseForEdX = apps.get_model("ccx", "CustomCourseForEdX")
+    Role = apps.get_model("django_comment_common", "Role")
+    Permission = apps.get_model("django_comment_common", "Permission")
+    db_alias = schema_editor.connection.alias
+    # This will need to be changed if ccx gets moved out of the default db for some reason.
+    if db_alias != 'default':
+        return
+
+    for ccx in CustomCourseForEdX.objects.using(db_alias).all():
```

Comment:
Why are we checking the `db_alias` explicitly here, instead of just saying `CustomCourseForEdX.objects.all()`?